amethyst_network/src/lib.rs

```diff
@@ -23,12 +15,22 @@ pub use crate::{
     server::{Host, ServerConfig, ServerSocketEvent},
 };
 
+use std::net::SocketAddr;
+use std::sync::mpsc::SyncSender;
```

Comment:
since you're moving these, could you group them into a `std::{net::SocketAddr, sync::mpsc::SyncSender};`? Thanks!

Reply:
Done. :)